x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/rule_actions/bulk_actions/bulk_edit_rules.cy.ts

```diff
@@ -133,16 +125,39 @@ describe('Detection rules, bulk edit', { tags: [tag.ESS, tag.BROKEN_IN_SERVERLES
     resetRulesTableState();
     deleteAlertsAndRules();
     cy.task('esArchiverResetKibana');
-    createRule(getNewRule({ name: RULE_NAME, ...defaultRuleData, rule_id: '1' }));
-    createRule(getEqlRule({ ...defaultRuleData, rule_id: '2' }));
-    createRule(getMachineLearningRule({ tags: ['test-default-tag-1', 'test-default-tag-2'] }));
-    createRule(getNewThreatIndicatorRule({ ...defaultRuleData, rule_id: '4' }));
-    createRule(getNewThresholdRule({ ...defaultRuleData, rule_id: '5' }));
-    createRule(getNewTermsRule({ ...defaultRuleData, rule_id: '6' }));
-
-    visitWithoutDateRange(SECURITY_DETECTIONS_RULES_URL);
-
-    waitForRulesTableToBeLoaded();
+    createRule(getNewRule({ name: RULE_NAME, ...defaultRuleData, rule_id: '1', enabled: false }));
+    createRule(
+      getEqlRule({ ...defaultRuleData, rule_id: '2', name: 'New EQL Rule', enabled: false })
+    );
+    createRule(
+      getMachineLearningRule({
+        name: 'New ML Rule Test',
+        tags: ['test-default-tag-1', 'test-default-tag-2'],
+        enabled: false,
+      })
+    );
+    createRule(
+      getNewThreatIndicatorRule({
+        ...defaultRuleData,
+        rule_id: '4',
+        name: 'Threat Indicator Rule Test',
+        enabled: false,
+      })
+    );
+    createRule(
+      getNewThresholdRule({
+        ...defaultRuleData,
+        rule_id: '5',
+        name: 'Threshold Rule',
+        enabled: false,
+      })
+    );
+    createRule(
+      getNewTermsRule({ ...defaultRuleData, rule_id: '6', name: 'New Terms Rule', enabled: false })
+    );
+
+    visitSecurityDetectionRulesPage();
+    disableAutoRefresh();
   });
```

Comment:
Here prebuilt rules are needed for some tests, but the actual package is installed from Fleet, which takes a long time and can create flakiness. Can you prevent the installation and mock a couple of rules?
```
      preventPrebuiltRulesPackageInstallation();

      const RULE_1 = createRuleAssetSavedObject({
        name: 'Test rule 1',
        rule_id: 'rule_1',
      });
      const RULE_2 = createRuleAssetSavedObject({
        name: 'Test rule 2',
        rule_id: 'rule_2',
      });

      createAndInstallMockedPrebuiltRules({ rules: [RULE_1, RULE_2] });
```

Reply:
Good point. I didn't pursue fixing everything in one PR 😅 just limit it to the common issues causing flakiness. While full prebuilt rules package installation doesn't seem to cause flakiness it makes sense to avoid installing 800+ prebuilt rules and as the changes a trivial I'll do it in this PR.